docroot/modules/custom/yukon_w3_custom/js/yukon_w3_custom.js

```diff
@@ -0,0 +1,9 @@
+jQuery("input[value='Soumettre un commentaire (this translation)']").click(function() {
+    if (jQuery(".path-form  .form-checkbox").prop("checked") == false){ 
+        alert("Please Checked Generate automatic URL alias");
+        jQuery(".path-form  .form-checkbox").focus();
```

Comment:
@lgontard Is the intention really to pester French translators with a pop-up/alert everytime the edit a node to check the automatic URL box? What if it's intentional? Seems like we're going to be annoying translators quite a bit.

Reply:
Moving the conversation over to https://github.com/ytgov/yukon-ca/issues/796